Compiler selection '4.02.3' is ambiguous.

When testing for `OCAML_VERSION=4.02`, I got the [failure message](https://travis-ci.org/Chris00/ocaml-odepack/jobs/447368227#L756):
```
[ERROR] Compiler selection '4.02.3' is ambiguous. matching packages: {
        ocaml-base-compiler.4.02.3, ocaml-system.4.02.3 }
The command "bash -ex .travis-opam.sh" exited with 2.
```

--- a/.travis-docker.sh
+++ b/.travis-docker.sh
@@ -61,7 +61,7 @@ fi
 case $opam_version in
     2.0.0)
       echo "RUN opam switch ${OCAML_VERSION} ||\
-          opam switch create ${OCAML_VERSION}" >> Dockerfile ;;
+          opam switch create ocaml-base-compiler.${OCAML_VERSION}" >> Dockerfile ;;
     *) ;;
 esac
 

